# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/gnome2/devhelp/compile.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./libegg/menu/egg-action.c.orig	2004-05-14 10:36:25.076328384 +0200
+++ ./libegg/menu/egg-action.c	2004-05-14 10:53:26.393064640 +0200
@@ -1,6 +1,7 @@
 #include <config.h>
 #include "egg-action.h"
 #include "../toolbar/eggtoolbutton.h"
+#include "../toolbar/eggtoolbar.h"
 #include "../util/eggintl.h"
 
 //#ifndef _
@@ -921,7 +922,7 @@
       /* gross hack!!! */
       for (tmp = GTK_TOOLBAR (button->parent)->children; tmp; tmp = tmp->next)
 	{
-	  GtkToolbarChild *tool_child = tmp->data;
+	  EggToolbarChild *tool_child = tmp->data;
 
 	  if (tool_child->widget == button)
 	    {
--- ./libegg/toolbar/eggtoolbar.c.orig	2004-05-14 10:32:25.233789992 +0200
+++ ./libegg/toolbar/eggtoolbar.c	2004-05-14 10:32:51.857742536 +0200
@@ -2570,7 +2570,7 @@
 
 GtkWidget*
 egg_toolbar_append_element (EggToolbar          *toolbar,
-			    GtkToolbarChildType  type,
+			    EggToolbarChildType  type,
 			    GtkWidget           *widget,
 			    const char          *text,
 			    const char          *tooltip_text,
@@ -2587,7 +2587,7 @@
 
 GtkWidget *
 egg_toolbar_prepend_element (EggToolbar          *toolbar,
-			     GtkToolbarChildType  type,
+			     EggToolbarChildType  type,
 			     GtkWidget           *widget,
 			     const char          *text,
 			     const char          *tooltip_text,
